mb-index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Michael Bond</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+  </head>
+  <body>
+  
+    <div class="container">
+      <div class="hero-image">
+        <p id="name">Mike <strong>Bond</strong></p>
+        <p class="hero-text">Web Designer</p>
+             <form id="button" action="resume.html"> <button class="hire-me">Hire me</button></form> 
+        <form=navigation>
+          <a class="navigation" href="index.html">Home</a>
+          <a class="navigation" href="resume.html">Resume</a>
```

Comment:
This link goes to an empty file -- you should move mb-resume.html to resume.html.